transit/rest_api/forms/shipment/shipment_details.py

```diff
@@ -119,6 +123,35 @@ class Meta:
         }
 
 
+class ShipmentAddCostsSerializerWrapper(serializers.ModelSerializer):
+    """
+        Utilizes ShipmentDetailsSerializer for purpose of bulk update of shipments costs.
+    """
+    _service = ShipmentAddCostsService()
+
+    id = serializers.IntegerField()
```

Comment:
Is this shipment ID? If that's bulk upload then ID should be list, if serilizer is for single shipment then I'm not sure if that's required. Shipment should be referenced under `details=True` (endpoint like api/shipment_cost/pk/add_cost_to_shipment, where pk is responsible for defining instance).